docs/source/release/v6.6.0/diffraction.rst

```diff
@@ -10,34 +10,60 @@ Powder Diffraction
 
 New features
 ############
-.. amalgamate:: Diffraction/Powder/New_features
+- Introduce an input parameter in :ref:`WANDPowderReduction <algm-WANDPowderReduction>` to specify that the input workspaces are from event filtering. In such a situation, the overall reduction time could be reduced significantly.
+- Added support for ILL D4C instrument, a liquid diffractometer. This adds a new algorithm responsible for the reduction stage: :ref:`D4ILLReduction <algm-D4ILLReduction>`, which handles calling the loader, correcting for dead time, bank position offsets, relative efficiency, and normalisation to monitor or time.
+- Increase the speed of the fourier filter in the POLARIS total scattering reduction by reducing the Rmax parameter used in the pair of (forward and backward) PDFFourierTransform calls and optimising the integration code inside :ref:`PDFFourierTransform v2 <algm-PDFFourierTransform-v2>`
 
 Bugfixes
 ############
-.. amalgamate:: Diffraction/Powder/Bugfixes
+- Change to use time average mean value of logs in :ref:`PDDetermineCharacterizations <algm-PDDetermineCharacterizations>`
+- Fix problem when calling focus twice in a reduction using the ISIS Powder diffraction scripts. The second run of focus was using a partially processed input file from the first focus resulting in some of the reduction steps (normalisation, absorption correction) were being run twice
```

Comment:
There's a leftover `were` in this sentence that I think was unintentional. While we're removing that, we could also consider adding a comma to break the sentence up a bit, so we'd end up with the below:

> The second run of focus was using a partially processed input file from the first focus, resulting in some of the reduction steps (normalisation, absorption correction) being run twice.